ksp_plugin/renderer.cpp

```diff
@@ -147,17 +150,20 @@ Renderer::RenderPlottingTrajectoryInWorld(
       from_plotting_frame_to_world_at_current_time =
           PlottingToWorld(time, sun_world_position, planetarium_rotation);
   for (auto it = begin; it != end; ++it) {
-    auto const& [time, degrees_of_freedom] = *it;
+    auto const& [t, degrees_of_freedom] = *it;
```

Comment:
Would a range-for loop work here?

Reply:
Yes, though we need `base::Range`. At some point we should start using `std::ranges`, and then this would be `std::ranges::subrange` (or rather, this should take a range instead of a pair of iterators).